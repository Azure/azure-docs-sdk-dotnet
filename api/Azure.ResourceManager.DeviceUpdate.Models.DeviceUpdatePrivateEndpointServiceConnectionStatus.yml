### YamlMime:NetType
type: struct
implementsWithMoniker:
- value: <xref href="System.IEquatable`1?alt=System.IEquatable&text=IEquatable" data-throw-if-not-resolved="True"/>&lt;<xref href="Azure.ResourceManager.DeviceUpdate.Models.DeviceUpdatePrivateEndpointServiceConnectionStatus?alt=Azure.ResourceManager.DeviceUpdate.Models.DeviceUpdatePrivateEndpointServiceConnectionStatus&text=DeviceUpdatePrivateEndpointServiceConnectionStatus" data-throw-if-not-resolved="True"/>&gt;
inheritancesWithMoniker:
- values:
  - <xref href="System.ValueType?alt=System.ValueType&text=ValueType" data-throw-if-not-resolved="True"/>
constructors:
- uid: Azure.ResourceManager.DeviceUpdate.Models.DeviceUpdatePrivateEndpointServiceConnectionStatus.#ctor(System.String)
  monikers:
  - azure-dotnet-preview
operators:
- uid: Azure.ResourceManager.DeviceUpdate.Models.DeviceUpdatePrivateEndpointServiceConnectionStatus.op_Equality(Azure.ResourceManager.DeviceUpdate.Models.DeviceUpdatePrivateEndpointServiceConnectionStatus,Azure.ResourceManager.DeviceUpdate.Models.DeviceUpdatePrivateEndpointServiceConnectionStatus)
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DeviceUpdate.Models.DeviceUpdatePrivateEndpointServiceConnectionStatus.op_Implicit(System.String)~Azure.ResourceManager.DeviceUpdate.Models.DeviceUpdatePrivateEndpointServiceConnectionStatus
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DeviceUpdate.Models.DeviceUpdatePrivateEndpointServiceConnectionStatus.op_Inequality(Azure.ResourceManager.DeviceUpdate.Models.DeviceUpdatePrivateEndpointServiceConnectionStatus,Azure.ResourceManager.DeviceUpdate.Models.DeviceUpdatePrivateEndpointServiceConnectionStatus)
  monikers:
  - azure-dotnet-preview
methods:
- uid: Azure.ResourceManager.DeviceUpdate.Models.DeviceUpdatePrivateEndpointServiceConnectionStatus.Equals(Azure.ResourceManager.DeviceUpdate.Models.DeviceUpdatePrivateEndpointServiceConnectionStatus)
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DeviceUpdate.Models.DeviceUpdatePrivateEndpointServiceConnectionStatus.ToString
  monikers:
  - azure-dotnet-preview
properties:
- uid: Azure.ResourceManager.DeviceUpdate.Models.DeviceUpdatePrivateEndpointServiceConnectionStatus.Approved
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DeviceUpdate.Models.DeviceUpdatePrivateEndpointServiceConnectionStatus.Pending
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DeviceUpdate.Models.DeviceUpdatePrivateEndpointServiceConnectionStatus.Rejected
  monikers:
  - azure-dotnet-preview
uid: Azure.ResourceManager.DeviceUpdate.Models.DeviceUpdatePrivateEndpointServiceConnectionStatus
commentId: T:Azure.ResourceManager.DeviceUpdate.Models.DeviceUpdatePrivateEndpointServiceConnectionStatus
namespace: Azure.ResourceManager.DeviceUpdate.Models
name: DeviceUpdatePrivateEndpointServiceConnectionStatus
fullName: Azure.ResourceManager.DeviceUpdate.Models.DeviceUpdatePrivateEndpointServiceConnectionStatus
nameWithType: Azure.ResourceManager.DeviceUpdate.Models.DeviceUpdatePrivateEndpointServiceConnectionStatus
assembliesWithMoniker:
- value: Azure.ResourceManager.DeviceUpdate.dll
packagesWithMoniker:
- value: Azure.ResourceManager.DeviceUpdate v1.0.0-beta.9
syntaxWithMoniker:
- lang: csharp
  values:
  - value: 'public readonly struct DeviceUpdatePrivateEndpointServiceConnectionStatus : IEquatable<Azure.ResourceManager.DeviceUpdate.Models.DeviceUpdatePrivateEndpointServiceConnectionStatus>'
- lang: fsharp
  values:
  - value: type DeviceUpdatePrivateEndpointServiceConnectionStatus = struct
- lang: vb
  values:
  - value: >-
      Public Structure DeviceUpdatePrivateEndpointServiceConnectionStatus

      Implements IEquatable(Of DeviceUpdatePrivateEndpointServiceConnectionStatus)
devLangs:
- csharp
- vb
- fsharp
monikers:
- azure-dotnet-preview
summary: The private endpoint connection status.
metadata:
  api_name:
  - Azure.ResourceManager.DeviceUpdate.Models.DeviceUpdatePrivateEndpointServiceConnectionStatus
  api_location:
  - Azure.ResourceManager.DeviceUpdate.dll
  topic_type:
  - apiref
  api_type:
  - Assembly
  f1_keywords:
  - Azure.ResourceManager.DeviceUpdate.Models.DeviceUpdatePrivateEndpointServiceConnectionStatus
  - Azure::ResourceManager::DeviceUpdate::Models::DeviceUpdatePrivateEndpointServiceConnectionStatus
  - DeviceUpdatePrivateEndpointServiceConnectionStatus
  helpviewer_keywords:
  - Azure.ResourceManager.DeviceUpdate.Models.DeviceUpdatePrivateEndpointServiceConnectionStatus struct [.NET]
  monikers:
  - azure-dotnet-preview
