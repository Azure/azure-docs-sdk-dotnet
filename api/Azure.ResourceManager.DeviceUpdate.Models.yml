### YamlMime:NetNamespace
classes:
- uid: Azure.ResourceManager.DeviceUpdate.Models.ArmDeviceUpdateModelFactory
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DeviceUpdate.Models.CheckNameAvailabilityContent
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DeviceUpdate.Models.CheckNameAvailabilityResponse
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DeviceUpdate.Models.ConnectionDetails
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DeviceUpdate.Models.DeviceUpdateAccountLocationDetail
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DeviceUpdate.Models.DeviceUpdateAccountPatch
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DeviceUpdate.Models.DeviceUpdateInstancePatch
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DeviceUpdate.Models.DeviceUpdatePrivateLinkServiceConnectionState
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DeviceUpdate.Models.DiagnosticStorageProperties
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DeviceUpdate.Models.GroupConnectivityInformation
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DeviceUpdate.Models.IotHubSettings
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DeviceUpdate.Models.PrivateEndpointUpdate
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DeviceUpdate.Models.PrivateLinkServiceConnection
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DeviceUpdate.Models.PrivateLinkServiceProxy
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DeviceUpdate.Models.RemotePrivateEndpoint
  monikers:
  - azure-dotnet-preview
structs:
- uid: Azure.ResourceManager.DeviceUpdate.Models.AuthenticationType
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DeviceUpdate.Models.CheckNameAvailabilityReason
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DeviceUpdate.Models.DeviceUpdateAccountLocationRole
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DeviceUpdate.Models.DeviceUpdatePrivateEndpointConnectionProvisioningState
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DeviceUpdate.Models.DeviceUpdatePrivateEndpointServiceConnectionStatus
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DeviceUpdate.Models.DeviceUpdateSku
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DeviceUpdate.Models.GroupIdProvisioningState
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DeviceUpdate.Models.PrivateEndpointConnectionProxyProvisioningState
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DeviceUpdate.Models.ProvisioningState
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DeviceUpdate.Models.PublicNetworkAccess
  monikers:
  - azure-dotnet-preview
uid: Azure.ResourceManager.DeviceUpdate.Models
commentId: N:Azure.ResourceManager.DeviceUpdate.Models
name: Azure.ResourceManager.DeviceUpdate.Models
fullName: Azure.ResourceManager.DeviceUpdate.Models
devLangs:
- csharp
monikers:
- azure-dotnet-preview
metadata:
  f1_keywords:
  - Azure.ResourceManager.DeviceUpdate.Models
  - Azure::ResourceManager::DeviceUpdate::Models
  - Models
  - DeviceUpdate.Models
  - DeviceUpdate::Models
  helpviewer_keywords:
  - Azure.ResourceManager.DeviceUpdate.Models namespace [.NET]
  monikers:
  - azure-dotnet-preview
