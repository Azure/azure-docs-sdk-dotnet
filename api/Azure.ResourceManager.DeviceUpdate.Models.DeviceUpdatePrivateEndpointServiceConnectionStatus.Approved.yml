### YamlMime:NetMember
type: property
members:
- returnsWithMoniker:
    type:
    - value: <xref href="Azure.ResourceManager.DeviceUpdate.Models.DeviceUpdatePrivateEndpointServiceConnectionStatus?alt=Azure.ResourceManager.DeviceUpdate.Models.DeviceUpdatePrivateEndpointServiceConnectionStatus&text=DeviceUpdatePrivateEndpointServiceConnectionStatus" data-throw-if-not-resolved="True"/>
  type: property
  uid: Azure.ResourceManager.DeviceUpdate.Models.DeviceUpdatePrivateEndpointServiceConnectionStatus.Approved
  commentId: P:Azure.ResourceManager.DeviceUpdate.Models.DeviceUpdatePrivateEndpointServiceConnectionStatus.Approved
  name: Approved
  fullName: Azure.ResourceManager.DeviceUpdate.Models.DeviceUpdatePrivateEndpointServiceConnectionStatus.Approved
  nameWithType: DeviceUpdatePrivateEndpointServiceConnectionStatus.Approved
  syntaxWithMoniker:
  - lang: csharp
    values:
    - value: public static Azure.ResourceManager.DeviceUpdate.Models.DeviceUpdatePrivateEndpointServiceConnectionStatus Approved { get; }
  - lang: fsharp
    values:
    - value: 'static member Approved : Azure.ResourceManager.DeviceUpdate.Models.DeviceUpdatePrivateEndpointServiceConnectionStatus'
  - lang: vb
    values:
    - value: Public Shared ReadOnly Property Approved As DeviceUpdatePrivateEndpointServiceConnectionStatus
  monikers:
  - azure-dotnet-preview
  summary: Approved.
  metadata: {}
uid: Azure.ResourceManager.DeviceUpdate.Models.DeviceUpdatePrivateEndpointServiceConnectionStatus.Approved*
commentId: Overload:Azure.ResourceManager.DeviceUpdate.Models.DeviceUpdatePrivateEndpointServiceConnectionStatus.Approved
namespace: Azure.ResourceManager.DeviceUpdate.Models
name: Approved
fullName: Azure.ResourceManager.DeviceUpdate.Models.DeviceUpdatePrivateEndpointServiceConnectionStatus.Approved
nameWithType: DeviceUpdatePrivateEndpointServiceConnectionStatus.Approved
assembliesWithMoniker:
- value: Azure.ResourceManager.DeviceUpdate.dll
packagesWithMoniker:
- value: Azure.ResourceManager.DeviceUpdate v1.0.0-beta.9
devLangs:
- csharp
- vb
- fsharp
monikers:
- azure-dotnet-preview
metadata:
  api_name:
  - Azure.ResourceManager.DeviceUpdate.Models.DeviceUpdatePrivateEndpointServiceConnectionStatus.Approved
  - Azure.ResourceManager.DeviceUpdate.Models.DeviceUpdatePrivateEndpointServiceConnectionStatus.get_Approved
  api_location:
  - Azure.ResourceManager.DeviceUpdate.dll
  topic_type:
  - apiref
  api_type:
  - Assembly
  f1_keywords:
  - Azure.ResourceManager.DeviceUpdate.Models.DeviceUpdatePrivateEndpointServiceConnectionStatus.Approved
  - Azure::ResourceManager::DeviceUpdate::Models::DeviceUpdatePrivateEndpointServiceConnectionStatus::Approved
  - Approved
  - DeviceUpdatePrivateEndpointServiceConnectionStatus.Approved
  - DeviceUpdatePrivateEndpointServiceConnectionStatus::Approved
  helpviewer_keywords:
  - DeviceUpdatePrivateEndpointServiceConnectionStatus.Approved property [.NET]
  - Approved property [.NET], struct DeviceUpdatePrivateEndpointServiceConnectionStatus
  monikers:
  - azure-dotnet-preview
